
Inactive Modules:
  1) defaultenv

The following have been reloaded with a version change:
  1) python/3.6 => python/3.8

2021-06-26 18:09:35.552219: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2021-06-26 18:09:35.552280: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2021-06-26 18:09:40.771419: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcuda.so.1
2021-06-26 18:09:41.299125: E tensorflow/stream_executor/cuda/cuda_driver.cc:328] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2021-06-26 18:09:41.299209: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (m7-16-3.rc.byu.edu): /proc/driver/nvidia/version does not exist
Traceback (most recent call last):
  File "gridsearch_5layers.py", line 51, in <module>
    (width1,width2,width3,width4,width5) = widths[modelnum] #TODO: change # outputs
IndexError: list index out of range
